README.md

```diff
@@ -112,3 +121,138 @@ make
 # T2 Hardware API
 
 When you `require('tessel')` within a script which is executed on Tessel 2, this loads a library which interfaces with the Tessel 2 hardware, including pins, ports, and LEDs, just like Tessel 1 ([Tessel 1 hardware documentation](https://tessel.io/docs/hardwareAPI)). The code for Tessel 2's hardware object can be found [here](https://github.com/tessel/t2-firmware/blob/master/node/tessel.js).
+
+## Ports and pins
+
+Tessel has two ports, A and B. They are referred to as `tessel.port['A']`.
+
+Tessel's ports can be used as module ports as in Tessel 1 (e.g. `accelerometer.use(tessel.port['B'])`), or used as flexible GPIO pins (e.g. `myPin = tessel.port['A'].pins[0]`).
+
+### Modules
+
+Tessel 2's module ports can be used with [Tessel modules](//tessel.io/modules) much as in [Tessel 1](http://start.tessel.io/modules).
+
+Here is an example of using the Tessel Climate module on Tessel's port B:
+
+```js
+var tessel = require('tessel');
+var climatelib = require('climate-si7020');
+var climate = climatelib.use(tessel.port['B']);
```

Comment:
Unless `climatelib` is used for anything else beyond the `use(...)` call, I'd argue that simplifying this initialization would be a welcome improvement: 

```js
var tessel = require('tessel');
var climate = require('climate-si7020').use(tessel.port.B);
```